tests/regression/02-base/08-glob_interval.c

```diff
@@ -1,4 +1,4 @@
-// PARAM: --set solver "'new'" --set ana.int.interval true
```

Comment:
Why not test the `new` solver?

Reply:
Same here, this is a fundamental test about intervals on global variables.

Reply:
I added a copy of this test with alternative `PARAM`.